packages/server/graphql/public/types/User.ts

```diff
@@ -9,6 +10,19 @@ const User: UserResolvers = {
     if (!domain || !isCompanyDomain(domain) || !isSuperUser(authToken)) return null
     return {id: domain}
   },
+  domains: async ({id: userId}, _args, {dataLoader}) => {
```

Comment:
-1 at the moment, if I create two orgs with the same user, I see the same domain twice. Should we make domains unique? 

![domains](https://user-images.githubusercontent.com/39854876/173824704-1d968310-2d2c-47bc-83a9-27314f7652e6.png)